### ISSUE TITLE
The loss and evaluation metrix get down together

### ISSUE THREAD

**Issue by Bigsealion** [MIC-DKFZ/nnUNet#838]
Hi Fabian,

At first, I was training a 3d nnUnet model, but it was terminated due to an accident.
Next, I run the following command to try to continue training:
`nnUNet_train 3d_fullres nnUNetTrainerV2 Task604_Seg 0 --npz -c`
Terminated epoch is 674, and the restart epoch is 650.

It runs well, but when the 822 epoch, there was a sudden sharp drop of the loss and evaluation metrix.

This is the training_log:
`2021-11-03 18:46:35.889826: 
epoch:  821 
2021-11-03 18:48:59.655825: train loss : -0.6542 
2021-11-03 18:49:11.657352: validation loss: -0.6396 
2021-11-03 18:49:11.677752: Average global foreground Dice: [0.8995, 0.7307] 
2021-11-03 18:49:11.678090: (interpret this as an estimate for the Dice of the different classes. This is not exact.) 
2021-11-03 18:49:12.921937: lr: 0.002115 
2021-11-03 18:49:12.922291: This epoch took 157.032387 s
 
2021-11-03 18:49:12.922360: 
epoch:  822 
2021-11-03 18:51:28.079007: train loss : -0.7710 
2021-11-03 18:51:39.730430: validation loss: -0.7770 
2021-11-03 18:51:39.731641: Average global foreground Dice: [0.8896, 0.0] 
2021-11-03 18:51:39.731805: (interpret this as an estimate for the Dice of the different classes. This is not exact.) 
2021-11-03 18:51:41.078411: lr: 0.002105 
2021-11-03 18:51:41.078916: This epoch took 148.156490 s`

![image](https://user-images.githubusercontent.com/23148550/140061893-a02ed5c1-65e8-4133-9273-4bd277c60f41.png)

1. Why the loss and evaluation metrix get down together, I think they should change in the opposite trend
2. I want to know why this changed so drastically, it's normal? Or is there something wrong with my command?

Thanks!

**Comment by FabianIsensee** [comment]
Hey,
not entirely sure what's going on but I have a hunch. Can you please train with `nnUNetTrainerV2_Loss_DiceCE_noSmooth` and see if the problem persists?
Best,
Fabian

**Comment by mrokuss** [comment]
Hey @Bigsealion 

To briefly answer your questions, this is indeed a weird behavior and certainly not the norm. The commands you used are correct (although they are different now in V2). Did you manage to reproduce this behavior and could the comment above resolve your issue? Otherwise I would close this issue.

Best,
Max
